### PR TITLE
Language tweaks in repo-summary README & GOVERNANCE

### PR DESCRIPTION
**Pull request by ebullient**

### CODE REVIEW

File: GOVERNANCE.md [hunk 1]
```diff
@@ -1,43 +1,35 @@
...
+### CF Council (CFC)
 
-See [Membership and Roles][membership] for more details.
+The  [CFC][] oversees strategic and operational decisions. To engage with the council, email the [`council` mailing list][CONTACTS.yaml] or initiate a GitHub discussion.
 
-## CF Advisory Board
+### Extended Governance Committee (EGC)
 
-The **CF Advisory Board** comprises representatives from supporting organizations. They offer strategic guidance to the CFC.
+The [EGC][] is a joint body formed from members of the CFC and a representative for each foundation projet. Itplays a crucial role in ensuring diverse project perspectives are included in foundation-wide decisions.
```
Comment (kenfinnigan) [inline GOVERNANCE.md:19]: s\Itplays\It plays

File: GOVERNANCE.md [hunk 2]
```diff
@@ -1,43 +1,35 @@
...
 
-For CFC elections, CF adopts a preference-based voting system facilitated by [Elekto](https://elekto.dev/).
+Organizations providing financial support to the CF appoint representatives to the [CF Advisory Board][cfab], which provides strategic guidance to the CFC.
 
-Every member, regardless of their role or contribution level, has an equal voice and is entitled to one vote in all decisions.
+### Decision Making
 
-## Record Keeping
+Decisions at CF follow the Lazy Consensus model, ensuring every member has a say. For details, see [Decision Making][cf-decision-making].
+### Record Keeping
```
Comment (kenfinnigan) [inline GOVERNANCE.md:32]: Do we have linting on the docs? We need a blank line before `### Record Keeping`

File: README.md [hunk 3]
```diff
@@ -17,7 +17,7 @@ Our vision is to create a dynamic and inclusive community where developers, cont
 
 ## Core Distinctions of CF
 
-Drawing inspiration from [The Codehaus][codehaus], CF offers the right balance of support without overwhelming governance.
+Drawing inspiration from [The Codehaus][codehaus], CF offers a unique balance between the foundation and prohects related to support governance.
```
Comment (kenfinnigan) [inline README.md:20]: s/prohects/projects

**Review by kenfinnigan — APPROVED** [review]
Couple of minor changes, substance is good

File: GOVERNANCE.md (hunk 1 above)
Comment (ebullient) [reply GOVERNANCE.md:19]: 🤦‍♀️

File: GOVERNANCE.md (hunk 2 above)
Comment (ebullient) [reply GOVERNANCE.md:32]: I usually have it in place in the editor. I'll look at a linter when we work on next big chunk.

**Review by kenfinnigan — APPROVED** [review]

**Pull request merged**